Makefile

```diff
@@ -143,13 +144,7 @@ e2e-docker: metering-src-docker-build
 	docker cp metering-e2e-docker:/out bin/e2e-docker-test-output
 	docker rm metering-e2e-docker
 
-vet:
-	go vet $(GO_PKG)/cmd/... $(GO_PKG)/pkg/...
```

Comment:
Can you remind why we're removing `make vet` as a verify check?

Reply:
My reasoning was that since we run "go test" elsewhere, all the code gets "go vet" run on it with a limited "high-confidence subset" of tests:

https://golang.org/pkg/cmd/go/internal/test/

Reply:
This is fine reasoning, but if I'm making a quick, one-off change locally, I'd rather run `make verify` before deploying Metering with those updates than `go test ./...`.

Reply:
Then it will be even faster to run `make verify` because it won't run `go vet` at that time?